box-ui/src/components/ts/FloatingBox.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Draggable from 'react-draggable';
+import '../css/FloatingBox.css';
+
+const FloatingBox = ({ children }: { children: any }) => {
+    return (
+        <div>
+            <Draggable>
+                <div className='pipWindow'>{children}</div>
+            </Draggable>
+        </div>
+    );
+};
```

Comment:
```suggestion
const FloatingBox = ({ props }: FloatingBoxProps) => {
    return (
        <div>
            <Draggable>
                <div className='pipWindow'>{props.children}</div>
            </Draggable>
        </div>
    );
};
```
```suggestion
const FloatingBox = ({ children }: { children: any }) => {
    return (
        <div>
            <Draggable>
                <div className='pipWindow'>{children}</div>
            </Draggable>
        </div>
    );
};
```
You should create a type for your props as it is done for other components in utils/props

Reply:
If you only need children you can only add FunctionComponent to make typescript happy and then get children from props